src/popup/popup.target.tsx

```diff
@@ -1,10 +1,23 @@
 import {createContext, forwardRef, type HTMLAttributes, type Ref, type ReactNode} from 'react';
 
-export const PopupTargetContext = createContext<string | Element | undefined>(undefined);
+interface ComplexConfig {
```

Comment:
I have decided to avoid creating yet another context and utilize "TargetContext" more. I think, semantically `cssPositioning` might belong to this context.

Please let me know your point of view